frontend/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_FEEDBACK_BUTTON=false
```

Comment:
let's make this `.env.development` and let's also add `env.production` so that we have separate environment files for the separate environments.

This will eventually be useful when we want some features in development but not production or vice versa. For now they will look the same since we don't want the feedback button in either!